.github/workflows/generator_container_slsa3.yml

```
@@ -48,12 +52,18 @@ on:
         required: false
         type: boolean
         default: false
+      continue-on-error:
+        description: "Prevents a workflow run from failing when a job fails. Set to 'true' to allow a workflow run to pass when a job fails."
+        required: false
+        type: boolean
+        default: false
 
 jobs:
   # detect-env detects the reusable workflow's repository and ref for use later
   # in the workflow.
   detect-env:
     outputs:
+      outcome: ${{ steps.final.outputs.outcome }}
```

nit: Does this not work?
```suggestion
      outcome: ${{ steps.detect.outcome }}
```

---

Should work. Shall I use you suggestion or keep as-is so that all the jobs have the same pattern and is easier to review / update?

---

I think its ok as is.